backend/.env.example

```
@@ -1,32 +1,53 @@
 REFRESH_TOKEN_EXPIRE_DAYS=30
 
-ENCRYPT_SECRET=your_encrypt_secret
+# Уровень логирования: DEBUG, INFO, WARNING, ERROR, CRITICAL
+LOG_LEVEL=WARNING
+# Логировать SQL-запросы (true/false)
+DB_ECHO=false
+
+# URL сайта (для ссылок в письмах и уведомлениях)
+SITE_URL=https://faunistica.ru
+# Разрешённые CORS-источники (через запятую)
+ALLOWED_ORIGINS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALLOWED_ORIGINS key should go before the SITE_URL key

(UnorderedKey)